ibis/backends/tests/test_register.py

```diff
@@ -96,6 +94,7 @@ def gzip_csv(data_dir, tmp_path):
         "trino",
     ]
 )
+@pytest.mark.notimpl(["druid", "exasol", "oracle"])
```

Comment:
I'm confuse about this, it looked like before we had support for this, why are we loosing this now?

Reply:
they were marked as not implemented at the begin of the file before: https://github.com/ibis-project/ibis/blob/main/ibis/backends/tests/test_register.py#L21

now I moved it to each test, because it no longer applies to all tests. 

> I'm confuse about this, it looked like before we had support for this, why are we loosing this now?